Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -285,11 +285,11 @@ public class HTTPClient {
         bootstrap.connect(host: address.host, port: address.port)
             .map { channel in
                 task.setChannel(channel)
-            }
-            .flatMap { channel in
-                channel.writeAndFlush(request)
-            }
-            .cascadeFailure(to: task.promise)
+        }
+        .flatMap { channel in
+            channel.writeAndFlush(request)
+        }
+        .cascadeFailure(to: task.promise)
```

Comment:
Can we remove these whitespace changes?

Reply:
I ran now with swiftformat ... i think they back to normal now ...